Runtime/Client/LootLockerServerRequest.cs

```diff
@@ -174,8 +183,11 @@ public class LootLockerResponse
             {
                 return LootLockerResponseFactory.Error<T>("Unknown error, please check your internet connection.");
             }
-            else if (serverResponse.errorData != null && !string.IsNullOrEmpty(serverResponse.errorData.code))
+            else if (serverResponse.errorData != null)
             {
+                if(!string.IsNullOrEmpty(serverResponse.errorData.code)) {
```

Comment:
The inside and outside of this if statement are the same, meaning it is pointless as it is now. Either move it back up or if you decide to go with the intended change in the rest of the code then this one is moot and the errorData != null is sufficient.